src/components/IconButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+
+import { Edit, Delete } from '../styles/data-list';
```

Comment:
With the new package regading the relative path , shouldn't it be '/styles/' instead of '../styles/' ?